spec/controllers/users_controller_spec.rb

```diff
@@ -10,6 +10,33 @@
     session[:user_id] = user.id
   end
 
+  describe '#create' do
+    let(:user_params) do
+      {
+        user: {
+          name: Faker::Name.name,
+          email: Faker::Internet.email,
+          password: 'Password123+',
+          password_confirmation: 'Password123+',
+          language: 'language'
+        }
+      }
+    end
+
+    it 'creates a current user if the new user is created' do
+      session[:user_id] = nil
+      create(:role, name: 'User') # Needed for admin#create
+      post :create, params: user_params
+      expect(session[:user_id]).to eql(User.last.id)
```

Comment:
Um, this seems to be okay but it somehow makes this test fragile since anything that can cause `ActiveRecord` to return the `user` record as the last in the `User` table will cause this to fail,

I'd go with something much safer maybe like:
```
 expect(session[:user_id]).to be_present
 expect(session[:user_id]).not_to (user.id)
```